modules/ocf_www/manifests/mod/fcgid.pp

```diff
@@ -0,0 +1,11 @@
+class ocf_www::mod::fcgid {
```

Comment:
do we have any way for users to restart their fastcgi processes?

it's probably not essential/blocking, but i suspect people will complain about the lack of it eventually

Reply:
I don't think there is a way currently, I'd like to play around with that a bit more and potentially set CGI execution time to a lower value to make the effects of this less annoying for people working on their websites live. (One of my TODOs in the PR description)

Were there many complaints before #3 was merged?

Reply:
Yeah. I can't find them in my inbox, but that's why we switched.

Reply:
Probably less of a concern now with apphosting -- though it will still affect individuals /shrug

Reply:
Yeah, I would like to make this not a problem for users if I can. I really wish that fastcgi was still maintained or that fcgid had a similar autoupdate feature, I've found a few threads online where people were asking for one but I couldn't find anything useful there. I'm hoping that lowering the time that scripts run by fcgid are alive for will help with this so that the docs can just say something like "wait for 30 seconds" or something would work decently well and still yield acceptable performance. After all, most websites aren't that popular (except dev-dev-vhost from health checks), so that helps.